cmd/main.go

```diff
@@ -105,22 +106,15 @@ func parseConfig() (Config, error) {
 	var configFileFlag string
 	var spireAPISocketFlag string
 	var expandEnvFlag bool
+	var logLevel zapcore.Level
```

Comment:
nit: this block of vars are for those provided by flags. I'd suggest moving logLevel down below, closer to the point of use.

Reply:
Sure, done!